src/lesson4/Task3.java

```diff
@@ -0,0 +1,39 @@
+package lesson4;
+
+import java.util.Scanner;
+
+public class Task3 {
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+
+        double array[] = new double[sc.nextInt()];
+
+
+        for (int i = 0; i < array.length; i++) {
+            array[i] = Math.random();
+
+            System.out.print(array[i] + " ");
+        }
+
+        double max = array[0];
+        double min = array[0];
+        double midl = 0;
+
+        for (int i = 0; i < array.length; i++) {
+            if (max < array[i])
+                max = array[i];
+            if (min > array[i])
+                min = array[i];
+            midl += array[i] / array.length;
```

Comment:
The avarage sum should be calculated outside the loop!
It's not correct to calculate it here